src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,138 @@
+/* eslint-disable max-len */
+import { useEffect, useState } from 'react';
+import classNames from 'classnames';
+import { useParams } from 'react-router-dom';
+import { Loader } from '../Loader';
+import { Person } from '../../types';
+import { getPeople } from '../../api';
+import { PersonLink } from '../PersonLink/PersonLint';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[] | null>(null);
```

Comment:
```suggestion
  const [people, setPeople] = useState<Person[]>([]);
```
usually for states with an array we set an empty array as the default value
it's easier to handle it below - no need to check `people` - just `people.length` etc.